pyhive/hive.py

```diff
@@ -212,7 +210,9 @@ def __init__(
                         # Password doesn't matter in NONE mode, just needs to be nonempty.
                         password = 'x'
 
-                def sasl_factory():
+
+                def get_sasl_client():
```

Comment:
let's make those functions top leve, not part of another class

Reply:
Will do

Reply:
done